common/src/main/scala/models/Notification.scala

```diff
@@ -66,6 +68,25 @@ object BreakingNewsNotification {
   implicit val jf = Json.format[BreakingNewsNotification]
 }
 
+case class NewsstandShardNotification(
+                                       id: UUID,
+                                       shard:Int
+                                     ) extends Notification {
+  override def title = ""
+  override def message: String = ""
+  override def sender: String = "newsstand-shard"
+  override def importance: Importance = Importance.Minor
+  override def withTopics(topics: Set[Topic]): Notification = this
```

Comment:
do you mind implementing that? It's short and would avoid spending time searching why it doesn't work if we suddenly decide to use it

Reply:
Nah after thinking about, don't bother